packages/richtext/src/utils/parse-tiptap-json.ts

```diff
@@ -0,0 +1,22 @@
+import { generateHTML } from '@tiptap/html';
+import { JSONContent, Extensions } from '@tiptap/core';
+import { debug } from '@sitecore-content-sdk/core';
+import StarterKit from '@tiptap/starter-kit';
+
+export const parseTiptapJSON = (content: JSONContent[], extensions: Extensions = []) => {
```

Comment:
Add jsdoc for the comment, also it would be good that we use starterkit by default

Reply:
We do use StarterKit by default. Or do you mean that `extensions` should override the default StarterKit?